src/main/java/io/micrometer/newrelic/transform/FunctionTimerTransformer.java

```diff
@@ -31,7 +31,7 @@ public FunctionTimerTransformer(TimeTracker timeTracker) {
 
     Summary summary =
         new Summary(
-            id.getName(),
+            id.getName() + ".summary",
```

Comment:
we very intentionally removed the suffixes since we're now only generating one metric, rather than a compound metric.